haskell-language-server.cabal

```diff
@@ -1992,12 +1917,11 @@ test-suite wrapper-test
   main-is:            Main.hs
 
 benchmark benchmark
-    import:           common-deps
+    import:           defaults, warnings, common-deps
     -- Depends on shake-bench which is unbuildable after this point
     if impl(ghc >= 9.5)
       buildable: False
     type: exitcode-stdio-1.0
-    default-language: Haskell2010
```

Comment:
You can remove -Wall -Wno-name-shadowing from benchmark now that you're importing warnings.